biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -1,5 +1,7 @@
 package aQute.bnd.build.model;
 
+import static java.util.Optional.of;
```

Comment:
I am not a fan of this static import. `of` in code is way too ambiguous. Stream has `of`. In Java 9, Map has `of`.